src/site/content/en/blog/gamepad/index.md

```diff
@@ -0,0 +1,347 @@
+  gamepad.vibrationActuator.playEffect('dual-rumble', {
+    // Start delay in ms.
+    startDelay: delay,
+    // Duration is ms.
+    duration: duration,
+    // The magnitude of the weak actuator (between 0 and 1).
+    weakMagnitude: weak,
+    // The magnitude of the strong actuator (between 0 and 1).
+    strongMagnitude: strong,
+  });
+};
+```
+
+### Integration with Permissions Policy
+
+The Gamepad API spec defines a
+[policy-controlled feature](https://w3c.github.io/webappsec-permissions-policy/)
+identified by the string `"gamepad"`. Its default `allowlist` is `"self"`.
+A document's permissions policy determines whether any content in that document is allowed
+to access `navigator.getGamepads()`.
+If disabled in any document, no content in the document will be allowed to use
+`navigator.getGamepads()`, nor will the `gamepadconnected` and
+`gamepaddisconnected` events fire.
+
+```html/2
+<iframe
+    src="index.html"
+    allow="gamepad"
+>
+</iframe>
+```
+
+## Demo
+
+A simple [gamepad tester demo](https://gamepad-demo.glitch.me/) is embedded below.
+The source code is available [on Glitch](https://glitch.com/edit/#!/gamepad-demo).
+Try the demo by connecting a gamepad via USB or Bluetooth and pressing any of its buttons
+or moving any of its axis.
+
+<div class="glitch-embed-wrap" style="height: 1000px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/gamepad-demo?path=index.html&previewSize=100"
+    title="gamepad-demo on Glitch"
+    allow="gamepad"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+## Bonus: Play Chrome dino on a Nintendo Switch
+
+The Nintendo Switch contains a
+[hidden browser](https://switchbrew.org/wiki/Internet_Browser#WifiWebAuthApplet),
+which essentially just serves for logging in to Wi-Fi networks behind a captive portal.
+The browser is pretty barebones and does not have a URL bar,
+but, once you have navigated to a page, it is fully usable.
+When doing a connection test in system settings, the Switch will detect that the captive portal
+is present and display an error for it when the response for
+[http://conntest.nintendowifi.net/](http://conntest.nintendowifi.net/)
+does not include the `X-Organization: Nintendo` HTTP header.
+We can make creative use of this by pointing the Switch to a DNS server
+that simulates to be a captive portal that then redirects to a search engine.
+
+1. Go to **System Settings** and then **Internet Settings** and
+   find the Wi-Fi network that your Switch is connected to. Tap **Change Settings**.
+1. Find the section with the **DNS Settings** and add
+   [45.55.142.122](http://45.55.142.122) as a new **Primary DNS**.
+   Note that this DNS server is *not operated by Google*
+   but a [third-party](https://www.switchbru.com/dns/), so proceed at your own risk.
+1. **Save** the settings and then tap **Connect to This Network**.
+1. The Switch will tell you that **Registration is required to use this network**. Tap **Next**.
```

Comment:
```suggestion
1. **Save** the settings and then tap **Connect to This Network**. The Switch will tell you that **Registration is required to use this network**. 
1. Tap **Next**.
```

Reply:
Steps always begin with a user action. Responses are always at the end of an action.

Reply:
Good call.